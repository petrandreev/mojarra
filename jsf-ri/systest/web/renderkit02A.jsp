<HTML>

    <%@ taglib uri="http://java.sun.com/jsf/html" prefix="h" %>
    <%@ taglib uri="http://java.sun.com/jsf/core" prefix="f" %>

       <f:use_faces>  
          <h:output_text id="simplelabel" value="Simple Label" /> 
       </f:use_faces>

</HTML>
