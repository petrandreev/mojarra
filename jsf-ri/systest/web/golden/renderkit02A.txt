<HTML>

    
    

         
          <span id="simplelabel">Simple Label</span> 
       

</HTML>
